chapters/pt/_toctree.yml

```
@@ -42,3 +42,6 @@
     title: Construindo um cartão para o modelo
   - local: chapter4/5
     title: Parte 1 completa!
+  - local: chapter4/6
+    title: Questionário de fim de capítulo
+    quiz: 3
```

Shouldn't this be a 4 here?
```suggestion
    quiz: 4
```